types/africastalking/package.json

```diff
@@ -0,0 +1,4 @@
+{
+    "private": true,
+    "type": "module"
```

Comment:
This package isn't a module; upstream is CJS and the d.ts file here says `export =`; you should delete this file.

Reply:
well noted @jakebailey